DogeNet.IdentityServer/Configuration.cs

```diff
@@ -0,0 +1,88 @@
+﻿using IdentityModel;
+using IdentityServer4;
+using IdentityServer4.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DogeNet.IdentityServer
+{
+    public static class Configuration
+    {
+        public static IEnumerable<ApiScope> ApiScopes => new List<ApiScope>
+        {
+            new ApiScope("DogeNetWebAPI", "Web API")
```

Comment:
All const strings should be moved to settings file

Reply:
![image](https://user-images.githubusercontent.com/77879337/142198496-4e7b3f1e-d1be-4550-820f-54323d712b6f.png)
![image](https://user-images.githubusercontent.com/77879337/142198524-2e5f4033-0c0e-4d7d-9db2-f45c147f830d.png)

@oleh-leobit like that?